privacy_policy/en/privacy-notice-product-temp-mail.md

```diff
@@ -0,0 +1,27 @@
+# AdGuard Temp Mail Privacy Notice
+
+At AdGuard Software Limited, we are committed to safeguarding your privacy and ensuring the protection of your data. This Privacy Notice explains the data we collect and process when you use AdGuard Temp Mail, our temporary email address generator service.
+
+## Data Collection and Processing
+
```

Comment:
I'd add in the very beginning, that the mailbox name is random and generated once when the user starts using the service. It is not associated with any IP address or the user information.